grc/start_application/__init__.py

```diff
@@ -77,6 +78,14 @@ def isFirstVisit():
         if form.validate_on_submit():
             if form.isFirstVisit.data == 'FIRST_VISIT' or form.isFirstVisit.data == 'LOST_REFERENCE':
                 try:
+                    applications_to_anonymise = Application.query.filter(
+                        Application.status == ApplicationStatus.STARTED,
+                        Application.email == session['validatedEmail']
+                    )
+
+                    for application_to_anonymise in applications_to_anonymise:
+                        anonymise_application(application_to_anonymise)
+
```

Comment:
Can you chat to Tor to double-check if this is the behaviour we want?

This will delete any duplicate applications as soon as someone clicks "First visit" or "Lost reference".

I can imagine someone clicking "Lost reference", then a few clicks later remembering where they'd written down there reference.

The other place we could do this is when someone submits an application - we could delete any **other** applications they have with that email address.